styx-common/src/main/java/com/spotify/styx/util/GcpUtil.java

```diff
@@ -38,8 +38,21 @@ public static boolean isPermissionDenied(GoogleJsonResponseException e) {
   }
 
   public static boolean isPermissionDenied(GoogleJsonError error) {
-    return Optional.ofNullable(error.get("status"))
-        .map("PERMISSION_DENIED"::equals)
+    return "PERMISSION_DENIED".equals(error.get("status"));
+  }
+
+  public static boolean isResourceExhausted(Throwable t) {
+    return t instanceof GoogleJsonResponseException
+           && isResourceExhausted((GoogleJsonResponseException) t);
+  }
+
+  public static boolean isResourceExhausted(GoogleJsonResponseException e) {
```

Comment:
I think the idea was to have them public as help methods (_might be used later but not now_ stuff).